tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -105,7 +105,7 @@ $aksedgeConfig = @"
         {
             "LinuxNode": {
                 "CpuCount": 8,
-                "MemoryInMB": 8192,
+                "MemoryInMB": 16384,
```

Comment:
16 for the cluster might be a blocker when the host only has 16, maybe 12 is a good balance? if not we might need to ask the user,  or calculate based on available memory.